core/evancore.vue.libs/src/i18n/de.json

```diff
@@ -172,6 +172,12 @@
       "desc": "Der dargestellte Bereich befindet sich momentan in der Entwicklung und wird bald verfügbar sein.",
       "title": "In Entwicklung"
     },
+    "user-callout": {
+      "account-settings": "Kontoeinstellungen",
+      "create-new-account": "Neuen Account erstellen",
```

Comment:
Is it really new account, not new Identity and Switch to other Identity?

Reply:
Hm, I simply followed the mockup. Please discuss changes to it with Daniel :p

Reply:
Good mention, it's wrong in the concept. Everything in this callout will be in context of a identity.